arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/CountDownLatch.kt

```diff
@@ -25,7 +25,11 @@ public class CountDownLatch(private val initial: Long) {
   public fun count(): Long = count.get()
   
   /** Await [count] to reach zero */
-  public suspend fun await(): Unit = signal.await()
+  public suspend fun await(): Unit =
```

Comment:
@nomisRev I had to change this methos because it was failing in JS. It seems that `CompletableDeferred` doesn't work correctly on that platform :(